cmd/graph.go

```diff
@@ -122,7 +122,7 @@ func graphFn(o *Options) error {
 	}
 
 	if o.Graph.GenerateDotFile {
-		return c.GenerateDotGraph()
+		return c.GenerateDotGraph(ctx)
```

Comment:
updated golancli locally and it had some more complaints about not using contexts in places, so fixed those while at this